src/app/modules/shared-components/components/message/message.component.ts

```diff
@@ -0,0 +1,27 @@
+import { Component, OnInit, Input } from '@angular/core';
+import { Message } from 'primeng/api';
+
+@Component({
+  selector: 'alg-message',
+  templateUrl: './message.component.html',
+  styleUrls: ['./message.component.scss']
+})
+export class MessageComponent implements OnInit {
+
+  @Input() type: 'success' | 'info' | 'error';
+  @Input() label: string;
+  @Input() closable = true;
+
+  msgs : Message[] = [];
+
+  constructor() { }
+
+  ngOnInit() {
+    this.msgs.push({
+      severity: this.type,
+      summary: '',
```

Comment:
This summary looks interesting (for our invitation case maybe as well), you should let the user specify an "summary" on MessageComponent as well.